addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js

```diff
@@ -204,8 +204,14 @@ export class CalendarCommonRenderer extends Component {
         el.dataset.eventId = event.id;
         el.classList.add("o_event", "py-0");
         const record = this.props.model.records[event.id];
+        const default_calendar_event_id = this.env.searchModel && this.env.searchModel.context.default_calendar_event_id;
 
         if (record) {
+            if (
+                default_calendar_event_id
+            ) {
+                this.openPopover(info.el, this.props.model.records[default_calendar_event_id]);
+            }
```

Comment:
I find it strange to "pollute" the web calendar view with default_calendar_event_id because the calendar application is not necessary installed even when the calendar view is available. 
Can't you use this.props.model.records[event.id]; in the popover ?
If not, I think the variable default_calendar_event_id should be named in a more generic way. It is confusing in this current form.

Reply:
Hello @arj-odoo ,
Thanks for review.
I made the changes please check this out!